### DISCUSSION TITLE
Changelog references wrong pull request.

### DISCUSSION THREAD

**Discussion by VictorioBerra** [twbs/bootstrap#30870] [General] [answered]
https://github.com/twbs/bootstrap/releases/tag/v4.5.0

> #29886: Close modal with keyboard=true & backdrop=static

Should be https://github.com/twbs/bootstrap/pull/29986

**Comment by mdo** [comment, marked as answer]
Fixed! Next time, open an issue for stuff like that please :).

**Reply by VictorioBerra** [reply]
I just wanted to test new discussions 😅
